src/themes/default/components/core/blocks/Header/AccountIcon.vue

```diff
@@ -2,10 +2,12 @@
   <div
     class="inline-flex relative dropdown"
     data-testid="accountButton"
-    @click.self="goToAccount"
+    @click.self="goToAccount();showMenu = true;"
     @keyup.enter="goToAccount"
     tabindex="0"
     role="button"
+    @mouseover="showMenu = true"
```

Comment:
isn't this solve issue?
I'm little bit worried about above syntax
```js
@click.self="goToAccount();showMenu = true;"
```
it opens modal window or changes route to account, but mouse pointer is still over, does he?

Reply:
This is a fallback for touch devices where we won't get mouseover event.

Adding click event we are making sure that on touch devices "User menu" will appear correctly

Reply:
Okay, thanks:)